SCD/src/EventAction.cc

```diff
@@ -200,7 +200,12 @@ void EventAction::EndOfEventAction(const G4Event *evt)
 			if ( config_var.doPFlow )
 			    pflow_obj.fill_cell_var();
 			trajectories.fill_var();
-			GraphConstructor graph_construct(cells_data_low.Cells_in_topoclusters, tracks_list_low.Tracks_list, trajectories.particle_to_track, graph_obj);
+
+			std::vector<float> _particle_dep_energies;
+			GraphConstructor graph_construct(cells_data_low.Cells_in_topoclusters, tracks_list_low.Tracks_list, trajectories.particle_to_track, graph_obj, &_particle_dep_energies);
+
+			trajectories.SetParticleDepEnergy( _particle_dep_energies );
```

Comment:
@pariRieck regarding the `particle_dep_energy` bug fix, this is the core of what was missing.  The `particle_dep_energy` branch was written but not filled with anything, because the vector was not properly passed to the GraphConstructor.cc. It must've gotten lost when moving away from super-resolution running.